docs/resources/schema-manager.md

```diff
@@ -0,0 +1,66 @@
+# DICOM Schema Manager
+
+#### What is it?
+Schema Manager is a command line app that upgrades the schema in your database from one version to the next. This requires migration scripts, starting with version 1, with each subsequent version providing both full and diff scripts. The typical migration script folder should look like this:
+- 1.sql
+- 2.sql
+- 2.diff.sql
+- 3.sql
+- 3.diff.sql
+- ...
+
+[DICOM Migration Scripts](https://github.com/microsoft/dicom-server/tree/main/src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations)
+
+------------
+
+#### How do you use it?
+DICOM Schema Manager currently has one command (**apply**) with the following options:
+
+| Option | Description |
+| ------------ | ------------ |
+| `-cs, --connection-string` | The connection string of the SQL server to apply the schema update. (REQUIRED) |
+| `-mici, --managed-identity-client-id` | The client ID of the managed identity to be used. |
+| `-at, --authentication-type` | The authentication type to use. Valid values are `ManagedIdentity` and `ConnectionString`. |
+| `-v, --version` | Applies versions between the current and specified version. |
+| `-n, --next` | Applies the next available version. |
+| `-l, --latest` | Applies versions between the current and latest version. |
+| `-f, --force` | The schema migration is run without validating the schema version. |
+| `-?, -h, --help` | Show help and usage information. |
+
+You can view the most up-to-date options by running the following command:
+`.\Microsoft.Health.Dicom.SchemaManager.Console.exe apply -?`
+
+Example command line usage:
+`.\Microsoft.Health.Dicom.SchemaManager.Console.exe apply --connection-string "server=(local);Initial Catalog=DICOM;TrustServerCertificate=True;Integrated Security=True" --version 20`
+
+`.\Microsoft.Health.Dicom.SchemaManager.Console.exe apply -cs "server=(local);Initial Catalog=DICOM;TrustServerCertificate=True;Integrated Security=True" --latest`
+
+------------
+
+#### Terminology
+Available version: Any version greater than or equal to the current version.
+Compatible version: Any version from SchemaVersionConstants.Min to SchemaVersionConstants.Max (inclusive).
+
+------------
+
+#### Important Database Tables
+SchemaVersion
+- This table holds all schema versions that have ben applied to the database.
+
+InstanceShema
+- Each DICOM instance reports the schema version it is at, as well as the versions it is compatible with, to the InstanceSchema database table.
+
+------------
+
+#### How does it work?
+
+Schema Manager runs through the following steps:
+1. Verifies all arguments are supplied and valid.
+2. Calls the [healthcare-shared-components ApplySchema function](https://github.com/microsoft/healthcare-shared-components/blob/20506ffba19905abe882812a25d74866d1e1dcb0/src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs#L53), which:
```

Comment:
This reference should probably point to the main branch, not a commit

Reply:
Good point, thanks! Fixed in latest commit.